src/en/nineanime/src/eu/kanade/tachiyomi/animeextension/en/nineanime/NineAnime.kt

```diff
@@ -447,8 +447,8 @@ class NineAnime : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
         ListPreference(screen.context).apply {
             key = PREF_DOMAIN_KEY
             title = "Preferred domain (requires app restart)"
-            entries = arrayOf("9anime.to", "9anime.gs", "9anime.pl", "9anime.id", "9anime.ph")
-            entryValues = arrayOf("https://9anime.to", "https://9anime.gs", "https://9anime.pl", "https://9anime.id", "https://9anime.ph")
+            entries = arrayOf("aniwave.to", "9anime.to", "9anime.gs", "9anime.pl", "9anime.id", "9anime.ph")
```

Comment:
Since all of the previous 9anime urls redirect to aniwave, there's no point in keeping them